operator/operator_test.go

```diff
@@ -178,7 +182,36 @@ func (e *environment) WaitPhase() {
 	for {
 		select {
 		case head := <-heads:
-			if e.enclaveParameters.IsLastPhaseBlock(head.Number.Uint64() - 1) {
+			if e.enclaveParameters.IsLastPhaseBlock(head.Number.Uint64()) {
+				log.Warn("Phase ended in block #", head.Number.Uint64())
```

Comment:
debug log

Reply:
The stupid `Warn` calls are only my way of Debug logging since i do not need to change the loglevel to see them :laughing:  
Will surely remove them.